platform/src/OutputPanel.js

```diff
@@ -1,5 +1,7 @@
+
 import { ModelPanel } from "./ModelPanel.js";
-import { language } from "./Playground.js";
+import { language } from "./Playground.js"
```

Comment:
Why did you remove the semicolon here?

Reply:
Agree this was unintentional so will revert.